src/components/GenderPicker.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import * as React from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+import {Picker} from '@react-native-picker/picker';
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    alignItems: 'center',
+  },
+  label: {
+    flexBasis: '25%',
+    fontSize: 16,
+    marginVertical: 3,
+  },
+  value: {
+    flexBasis: '75%',
+    fontSize: 16,
+    marginVertical: 3,
+  },
+  pickerItem: {
+    fontSize: 16,
+  },
+});
+
+const GenderPicker = ({
+  label,
+  value,
+  onValueChange,
+}: {
+  label: string,
+  value: ?string,
+  onValueChange: ?(itemValue: string | number, itemIndex: number) => mixed,
```

Comment:
Could you explain why you return `=> mixed`, why don't use `=> void`

Reply:
From my perspective, this component is a wrapper of Picker component. Hence, I took the signature of the onValueChange prop of Picker component, and put it here